nodes/object_nodes/closest_point_on_mesh2.py

```diff
@@ -43,7 +43,7 @@ def sv_init(self, context):
         si('SvVerticesSocket', "point").use_prop = True
         si('SvStringsSocket', "max_dist").prop_name = "Mdist"
 
-        so('SvStringsSocket', "succes")
+        so('SvStringsSocket', "success")
```

Comment:
Names of sockets should definitely not be changed without creating MK next node version.

Reply:
Reverted in 27d9b24